app/views/movies/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<h1>Movie Show Page</h1>
+
+  <h3><br><%= "Title: #{@movie.title}" %></h5>
+  <%= "Creation Year: #{@movie.creation_year}" %>
+  <%= "Genre: #{@movie.genre}" %>
+ 
+  <h4><br><%= "Actors in #{@movie.title}" %></h4>
+  <% @movie.youngest_first.each do |actor| %>
+        <%= "Name: #{actor.name}" %>
+    <br>
+  <% end %>
+
+  <h4><%= "Actors Average Age: #{@actors_average_age}" %></h4>
+<br>
+
+  <section id="add-actor">
+  <h3>Add an actor to this movie</h3>
+  <%= form_with url: "/movie_actors", method: :post, data: { turbo: false } do |form| %>
```

Comment:
Awesome RESTful route here! You might have run into trouble trying to access the movie_id because it wasn't passed as a hidden field, so keep in mind that you could've used this route as well, which is also RESTful: "/movies/:movie_id/actors"